src/libs/ajax/leonardo/models/disk-models.ts

```diff
@@ -108,9 +107,10 @@ export const googlePdTypes: Record<PDLabels, GooglePdType> = {
     regionToPricesName: 'monthlySSDDiskPrice',
   },
 };
-export type DecoratedPersistentDisk = {
+export type PersistentDisk = {
   diskType: GooglePdType;
-} & Omit<PersistentDisk, 'diskType'>;
+} & Omit<RawPersistentDisk, 'diskType'>;
```

Comment:
just curious, what does the `Omit<>` do here?

Reply:
My understanding is that it removes the original diskType from the object so it can be replaced by the GooglePdType